overlay.nix

```diff
@@ -2,9 +2,19 @@ self: super:
 
 with super.lib;
 
-rec {
-  hello = self.callPackage ./pkgs/hello { };
+let replitPackages = rec {
+  # These packages will hide packages in the top level nixpkgs
+  override = {
+    hello = self.callPackage ./pkgs/hello { };
```

Comment:
Do we want `hello` in the top level? Maybe we can remove it altogether now that we have some other packages.